blocks/hero/hero.css

```diff
@@ -14,6 +14,26 @@ main .section.hero-container {
   padding: 2.625rem 0;
   overflow: hidden;
 
+  & picture.hero-background.mobile-only {
```

Comment:
You can probably just move the declaration in the 992px breakpoint to here, then swap out the mobile/desktop display state to based on breakpoint. That way you only declare the picture/image css once.